src/pages/ProjectControl/ProjectControl.tsx

```diff
@@ -0,0 +1,5 @@
+const ProjectControl: React.FC = () => {
```

Comment:
NIT: Can export directly and ideally we use typography, but not critical since I know this is a placeholder